docker/mms_app_gpu.conf

```diff
@@ -20,13 +20,18 @@ optional
 --metrics-write-to
 optional
 
+--num-GPUs
+optional
```

Comment:
What would we default to when this is optional? For now, I think we can hard code it to 1 to avoid  last minute churn.